lib/index.js

```diff
@@ -31,15 +31,15 @@ internals.implementation = function (server, options) {
       if (!token && request.auth.mode === 'optional') {
         return reply.continue({ credentials: {} });
       }
-      
+
       if (!token) {
-        return reply(Boom.unauthorized('Missing auth token'));
+        return reply(Boom.unauthorized(null, 'Token'));
```

Comment:
First parameter of `Boom.unauthorized` is the `error message`
https://github.com/hapijs/boom#boomunauthorizedmessage-scheme-attributes
We could do:
```js
return reply(Boom.unauthorized('Missing JWT Auth token', 'Token'));
```

Is there a reason you want to return `null` as the first argument?